src/esp/assets/ResourceManager.cpp

```diff
@@ -1289,24 +1289,32 @@ bool ResourceManager::loadRenderAssetIMesh(const AssetInfo& info) {
       (fileImporter_->openFile(filename) && (fileImporter_->meshCount() > 0)),
       Cr::Utility::formatString("Error loading semantic mesh data from file {}",
                                 filename));
-
-  const auto meshCount = fileImporter_->meshCount();
-  if (meshCount == 1) {
-    // only one mesh, treat as before
-    meshData = fileImporter_->mesh(0);
+  // Transform meshData by reframing rotation.  Doing this here so that
+  // transformation is caught in OBB calc.
+  const Magnum::Matrix4 reframeTransform = Magnum::Matrix4::from(
+      Magnum::Quaternion(info.frame.rotationFrameToWorld()).toMatrix(),
+      Magnum::Vector3());
```

Comment:
Normally we process the reframing transform (where we specify new axes for up/fwd) after the MeshMetadata is built (where we apply it to the MeshMetadata root), but we need to apply this transformation here so that the vertices are appropriately transformed to calculate accurate semantic OBBs